lib/plugins/inspec-parallel/lib/inspec-parallel/runner.rb

```diff
@@ -34,6 +35,10 @@ def run
           cleanup_child_processes
           sleep 0.1
         end
+
+        # Requires renaming operations on windows only
+        # Do Rename and delete operations after all child processes have exited successfully
```

Comment:
There is an unfortunate effect that if you interrupt (Ctl-C) the parallel process controller, the files do not get renamed, so they end up being named with the timestamp. That would be very frustrating. Could we improve that? Can we call the rename hook from the control-c interrupt?

Reply:
Done, but the way I have handled, it also captures interrupt exceptions